### DISCUSSION TITLE
Improve Next JS Performance

### DISCUSSION THREAD

**Discussion by surjithctly** [vercel/next.js#20295] [Help]
I ran a lighthouse test of my first (yay!) Next JS landing page. However, it shows a poor performance score. 

See the report: https://lighthouse-dot-webdotdevsite.appspot.com//lh/html?url=https%3A%2F%2Fweb3forms.com%2F

![image](https://user-images.githubusercontent.com/1884712/102599410-6fd86900-4143-11eb-872d-6a3f253c53fd.png)

All suggestions are based on Next JS. So how can I improve those? Any ideas? Looking for a score above 90%. 

Thanks.

**Comment by olaj** [comment]
I have the same issue with preload warning, and also unused JavaScript. I suppose unused Javascript is hard to get rid of but im going crazy about that preload warning.

I haven't really gotten any closer to get rid of it since posting this.

https://github.com/vercel/next.js/discussions/18287
